hand.py

```diff
@@ -0,0 +1,160 @@
+import cv2
+import mediapipe
+import pyautogui
+import random
+import os
+import pyscreeze
+
+# Initialize Hand tracking module
+capture_hands = mediapipe.solutions.hands.Hands()
+drawing = mediapipe.solutions.drawing_utils
+
+# Get screen dimensions
+screen_height, screen_width = pyautogui.size()
+
+window_width = int(screen_width * 1.0)
+window_height = int(screen_height * 0.4)
+
+# Open webcam
+cam = cv2.VideoCapture(0)
```

Comment:
Consider adding error handling for webcam initialization to handle cases where the webcam is not accessible or not found [important]

- Generated by [LGTM](https://lgtm.pro)